env_test.go

```diff
@@ -153,6 +153,16 @@ type NestedStruct struct {
 	NestedVar string `env:"nestedvar"`
 }
 
+func TestIssue245(t *testing.T) {
+	t.Setenv("NAME_NOT_SET", "")
+	type user struct {
+		Name string `env:"NAME_NOT_SET" envDefault:"abcd"`
+	}
+	cfg := user{}
+	isNoErr(t, Parse(&cfg))
+	isEqual(t, cfg.Name, "abcd")
```

Comment:
Should not this be equal to `""` or whatever is used in L155?

Reply:
no, that was the previous behavior, that this PR proposes to change